packages/extension-browser/src/devtools/views/controls/summary.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+import { ElementProps } from '../../utils/types';
+
+import * as styles from './summary.css';
+
+export type Props = ElementProps<'summary'>;
+
+/**
+ * Summary configuration inputs with common styles.
+ */
+
+const Summary = ({ className, ...props }: Props) => {
+    return (
+        <summary className={styles.summary}{...props} />
```

Comment:
This currently will drop any `className` passed into the component. We can use the same approach as `label.tsx` to preserve it (also assuming the rename of the base `className` to `root`):
```suggestion
        <summary className={`${styles.root} ${className || ''}`} {...props} />
```